app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<section class="mechanic_info">
+  <h1> Mechanic <%= @mechanic.name %> </h1>
+  <p> Years of Experience: <%= @mechanic.years_experience %> </p>
+</section>
+
+<section class="mechanic_rides">
+  <h1> Rides <%= @mechanic.name %> is currently working on </h1>
+  <ul>
+    <% @mechanic.rides.each do |ride| %>
+      <li><p> <%= ride.name %> </p></li>
+    <% end %>
+  </ul>
+</section>
+
+<section class="new_ride">
+  <h1> Add a Ride for <%= @mechanic.name %> </h1>
+    <%= form_with url: "/mechanics/#{@mechanic.id}/rides", method: :post, data: {turbo: false} do |form| %>
+      <%= form.label :ride_id, "Add Ride by ID" %>
+      <%= form.text_field :ride_id %>
+      <%= form.submit "Submit" %> 
+    <% end %>
+</section>
```

Comment:
No feedback is required for this page, it's all correct! Perfect verb and path for your form.